gradle-jdks-setup/build.gradle

```diff
@@ -21,10 +22,23 @@ dependencies {
     integrationTestImplementation 'org.assertj:assertj-core'
 }
 
+task fatJar(type: Jar) {
+    dependsOn tasks.jar, ':gradle-jdks-setup-common:jar'
+    manifest {
+        attributes 'Main-Class': 'com.palantir.gradle.jdks.setup.GradleJdkInstallationSetup'
+    }
+    archiveBaseName = 'gradle-jdks-setup-all'
+    duplicatesStrategy = DuplicatesStrategy.FAIL
+    from { configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) } }
+    with jar
+}
+
 tasks.withType(JavaCompile) {
     options.errorprone.disable 'PreferSafeLoggableExceptions'
 }
 
 tasks.integrationTest {
     environment("PROJECT_VERSION", project.version)
 }
+
+tasks.build.dependsOn tasks.fatJar
```

Comment:
How are we going to ship the fat jar to people? I guess we don't have the code in the plugin to do this yet (?), but I imagine we're either going to include the fat jar as a resource or download it?

Reply:
Correct, we don't have yet the code in the plugin to do this yet. 
I think it needs to be distributed alongside "./gradle-jdks.setup.sh" and the "gradle/" files. I was thinking of downloading it (similar how the wrapper.jar gets added to a project). I'll do this in the next PR.